packages/hyperion-autologging/src/ALInteractableDOMElement.ts

```diff
@@ -157,7 +179,7 @@ let installHandlers = () => {
   IReactComponent.onReactDOMElement.add((_component, props: ReactComponentObjectProps) => {
     if (props != null) {
       UIEventNames.forEach(event => {
-        if (props[SYNTHETIC_EVENT_HANDLER_MAP[event]] != null) {
+        if (props[SYNTHETIC_EVENT_HANDLER_MAP[event]] != null && !isListenerNoop(props[SYNTHETIC_EVENT_HANDLER_MAP[event]])) {
```

Comment:
read `props[SYNTHETIC_EVENT_HANDLER_MAP[event]]` into a separate variable to avoid reading it twice (this is two loads).